devicon.json

```diff
@@ -7494,6 +7494,24 @@
         "color": "#f37036",
         "aliases": []
     },
+    
+    {
+        "name": "powerpages",
+        "altnames": [],
+        "tags": [
+            "software"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
```

Comment:
In plain version logo should be black ? And have the same path ?

Reply:
@Yashh56 it should have no fill color (which makes it black in the preview).
It can have the same path, but it needs to be a united path. So only one <path /> element

Reply:
> Hey @Snailedlt I just convert all the paths to in one path but the image get transparent white like empty screen what should I do ?

Can you upload the icon in a comment so that I can see the code please?

Reply:
I'd need see the code to be able to help

Reply:
@Yashh56 This SVG is quite complicated. I'm not able to easily convert it to plain or line versions... Best to ask help from someone better at SVG editing than me. 
@lunatic-fox, @weh, @ConX and @kilianpaquier are really good at this stuff. So let's wait for one of them to respond :)

Reply:
When you open this in Inkscape it shows the image